src/components/PreviousJobs.jsx

```diff
@@ -13,12 +13,16 @@ const PreviousJobs = () => {
 
   const getJobsAhead = () => {
     api
-      .get(`/jobs`)
+      .get(`/jobs?limit=30`)
```

Comment:
Test was failing because this URL changed and so was no longer being mocked. I have updated.